iap/validate.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iap
+
+// [START iap_validate_jwt]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"google.golang.org/api/idtoken"
+)
+
+// validateJWTFromAppEngine validates a JWT found in the
+// "x-goog-iap-jwt-assertion" header.
+func validateJWTFromAppEngine(w io.Writer, iapJWT, projectNumber, projectID string) error {
+	// iapJWT := "base64.encoded.jwt"
```

Comment:
Make this a fake value? Could add a comment after the value saying where to find that value. Maybe we should add an HTTP response as an argument to show pulling it from the header?

Reply:
Added a comment to the comment for now. If you don't think this is clear we can add a parameter though. Between this example and the Go doc above the function, I think the point is made.